.github/workflows/unittest_ci_cpp.yml

```diff
@@ -0,0 +1,82 @@
+name: C++ Unit Test CI
```

Comment:
Shall we just run it as a part of the unittest_ci_cpu.yml / unittest_ci.yml ?

Reply:
I've moved this into unittest_ci_cpu.yml.

Reply:
@zhuzilin  we should run this in both unittest_ci_cpu as well as the binary push flows https://github.com/pytorch/torchrec/actions